### PR TITLE
Api

### PR DESCRIPTION
**Pull request by LuisDavidRodriguez**
# Changes 
- Get the books from the API
- Add Books 
- Remove books

> Hi dear reviewer I'm submitting with some console logs please ignore them this is intended because I want to use this project has a guidance for future projects, that console logs will help me to know what can I receive from the API

image

image

Mi video-12

### CODE REVIEW

File: src/redux/books/apiBook.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+const UNIQ_ID = 'IUmpICCqFqLeGrbCEOb9';
+const BASE_URL = `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${UNIQ_ID}/books/`;
+
+export const getBooks = createAsyncThunk(ADD_ARR_BOOK, async () => {
+  const response = await fetch(BASE_URL);
+  const data = await response.json();
+  // console.log('asynk');
+  // console.log(data.item1);
+  // console.log(data);
```
Comment (Nemwel-Boniface) [inline src/redux/books/apiBook.js:12]: - [ ] Kindly make sure that you remove all commented-out/ unwanted pieces of code from your code base. It makes it look messy and cluttered.

File: src/redux/books/books.js [hunk 2]
```diff
@@ -1,35 +1,38 @@
...
       return state;
   }
 };
 
 // action creators
+// this one was before use the createAsyncTunk
+// const addArrBook = (arr) => ({
+//   type: ADD_ARR_BOOK,
+//   arrBook: arr,
+// });
```
Comment (Nemwel-Boniface) [inline src/redux/books/books.js]: - [ ] Kindly make sure that you remove all commented-out/ unwanted pieces of code from your code base. It makes it look messy and cluttered.

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @LuisDavidRodriguez        :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job working on your project so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by LuisDavidRodriguez** [comment]
@Nemwel-Boniface hi there , my friend I wrote you a message in the pull request, I did it intended because I want to keep that console log because I will need this project has a guidance and I want to see there the consoles! just for now, because I want to study it! blease skiip that part!

**Comment by LuisDavidRodriguez** [comment]
@Nemwel-Boniface  I know that is not a good practice, I have not done so in previous projects but in this one I need it
Also please you could have just keep them as Optional not required u.u

**Review by m05e5 — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @LuisDavidRodriguez  🔥 

Good job on implementing the requirements. I understand, you can keep working with the `console.log`. Just keep in mind that in the future when developing apps its better not to have `console.log` in your code when merging to dev.  Well Done.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

goodjob


Now you can merge this branch 🚀 

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

_m05e5_

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**